### PR TITLE
[test] Disable a test in StringIndex

### PR DESCRIPTION
**Pull request by Azoy**
Disable this test currently due to rdar://99627624.

### CODE REVIEW

**Comment by Azoy** [comment]
@swift-ci please smoke test

File: test/stdlib/StringIndex.swift [hunk 1]
```diff
@@ -1060,7 +1060,8 @@ suite.test("String.replaceSubrange index validation")
   }
 }
 
-suite.test("Substring.removeSubrange entire range") {
+suite.test("Substring.removeSubrange entire range")
+.xfail(.always("Disabled due to rdar:\/\/99627624")) {
```
Comment (rintaro) [inline test/stdlib/StringIndex.swift:1064]: Isn't this invalid escape sequences?

Comment (Azoy) [reply test/stdlib/StringIndex.swift]: Oh my editor turned this (//) into a comment and I thought it was illegal. Thanks 👍

**Comment by Azoy** [comment]
@swift-ci please smoke test

**Review by rintaro — APPROVED** [review]

**Comment by Azoy** [comment]
@swift-ci please smoke test

**Pull request merged**